### PR TITLE
DOCS Audit event attributes in new format

### PR DESCRIPTION
**Pull request by albertzaharovits**
This is the spring cleaning in the audit event attributes docs.
A lot has changed and the docs have been left behind.
Some of the documented updates:
* Structured audit entries https://github.com/elastic/elasticsearch/pull/31931
* X-Opaque-Id
* Roles list

### CODE REVIEW

File: x-pack/docs/en/security/auditing/event-types.asciidoc [hunk 1]
```diff
@@ -43,189 +45,458 @@ multiple audit events. The following is a list of the events that can be generat
...
+
+The new format is output to the `<clustername>_audit.log` file.
+The audit entries are formatted as flat JSON documents (that is to say, no
+nested objects), one per line. Hence, the attribute names are JSON keys and they
+follow a dotted name syntax. Any attributes that lack a value (`null`) are not
+output.
+
+The following list shows attributes that are common to all audit events.
+Their names and values are analogous to those in the deprecated `logfile` or
+`index` output formats. However, it is expected that the formats will evolve
```
Comment (lcawl) [inline x-pack/docs/en/security/auditing/event-types.asciidoc]: Is it correct that the `index` output format is deprecated?  If not, maybe change this phrase to "... the index or deprecated logfile output format..."

**Review by lcawl — APPROVED** [review]
I have one question, otherwise LGTM.  I verified that it builds successfully.

**Comment by albertzaharovits** [comment]
run gradle build tests

**Review by jaymode — APPROVED** [review]

**Comment by albertzaharovits** [comment]
run gradle build tests 1
and
run gradle build tests 2

**Pull request merged**